blocks/page_list/templates/worldskills_news_list.php

```diff
@@ -3,8 +3,18 @@
 $th = Loader::helper('text');
 
 $thumbnailType = \Concrete\Core\File\Image\Thumbnail\Type\Type::getByHandle('landscape');
+
+$app = \Concrete\Core\Support\Facade\Application::getFacadeApplication();
+$canonical = $app->make(Concrete\Core\Url\SeoCanonical::class);
+$tag = $canonical->getPathArguments()[1];
 ?>
 
+<h1 class="ws-h-icon h2"><?php
+    if (isset($pageListTitle) && $pageListTitle) echo h($pageListTitle); else echo 'News';
+    if (isset($tag) && $tag) echo ' — ';
+    echo $tag; ?>
```

Comment:
You should use `h($tag)` here to escape the output - otherwise we have an XSS vulnerability.  I'm also not a huge fan of one line `if` statements... can you change this to:

```php
if (isset($tag) && $tag) {
    echo ' — ';
    echo h($tag);
}
```